EFCore/MAUI/Readme.md

```diff
@@ -212,15 +236,15 @@ This example demonstrates how to use our [Secured WebAPI](https://docs.devexpres
         return new ModuleUpdater[] { new DatabaseUpdate.Updater(objectSpace, versionFromDB),predefinedReportsUpdater };
     }
    ```
-## Step 6. [Use the Swagger UI to test the WebApi project](https://docs.devexpress.com/eXpressAppFramework/404281/backend-web-api-service/test-the-web-api-with-swagger-postman) custom and build-in endpoints. 
+## Step 7. [Use the Swagger UI to test the WebApi project](https://docs.devexpress.com/eXpressAppFramework/404281/backend-web-api-service/test-the-web-api-with-swagger-postman) custom and build-in endpoints. 
```

Comment:
Please fix naming everywhere: 
1. WebAPI > Web API or Web API Service (with a space)
2. MAUI > .NET MAUI (this is how Microsoft calls it everywhere in their docs and materials)
3. a specific Business object > a specific business object (a small case, because it is not a name - fix in multiple locations). We use capital letters only when we refer to a specific Item Template name or another name.